python3-pyside6.spec

```diff
@@ -28,14 +28,14 @@
 # Package Description
 
 Name:           python3-pyside6
-Version:        6.6.2
+Version:        6.6.3.1
 Release:        1%{?dist}
 Summary:        Python bindings for Qt 6, including the Qt 6 library, backported from the official Python wheels
 License:        LGPL-3.0-only OR (GPL-2.0-only OR GPL-3.0-or-later) AND GPL-2.0-only AND GPL-3.0-only WITH Qt-GPL-exception-1.0
-Source0:        https://download.qt.io/official_releases/QtForPython/pyside6/PySide6-%{version}-cp38-abi3-manylinux_2_28_x86_64.whl
-Source1:        https://download.qt.io/official_releases/QtForPython/pyside6/PySide6_Addons-%{version}-cp38-abi3-manylinux_2_28_x86_64.whl
-Source2:        https://download.qt.io/official_releases/QtForPython/pyside6/PySide6_Essentials-%{version}-cp38-abi3-manylinux_2_28_x86_64.whl
-Source3:        https://download.qt.io/official_releases/QtForPython/pyside6/shiboken6-%{version}-cp38-abi3-manylinux_2_28_x86_64.whl
+Source0:        https://download.qt.io/snapshots/ci/pyside/6.6.3.1/RC2/PySide6-%{version}-cp38-abi3-manylinux_2_28_x86_64.whl
```

Comment:
I'm sure you know this, but I didn't see it mentioned elsewhere, the URLs for these wheels are marked as RC2 snapshots...are they real releases? And if not, are they still safe to use? A comment (probably in the spec file) addressing that we're intentionally using "RC2" builds would be nice.

Reply:
Yeah, I admit, I don't like it as well. Unfortunately, there's no official PySide6 6.6.3.1 wheel in their releases page. My best guess is that they were in a hurry (hence the weird version) and didn't copy these artifacts to their official releases page.

Still, the hashes of these RC2 wheels match the hashes of the PyPI wheels, which will be eventually installed in our Windows and macOS users. So, I guess we can live with that.

I had written the above down in the corresponding commit (9a498235fbebc1045a45c9e508bcada4965bf3d3). I've also sent a fixup commit with a comment, as you proposed: 50cfae976964d82e67b883a16e236a418750ceb4

Is it more clear now?

Reply:
Yep, thanks!